src/components/tweet.js

```diff
@@ -44,15 +45,40 @@ export default class Tweet extends React.Component {
     return this.props.tweet.entities.media.map((media) => {
       if (media.type === 'photo') {
         return (
-          <a href={media.expanded_url} key={media.id_str} target='_blank'>
-            <img className='tweet_media' src={media.media_url} />
+          <a href="javascript:void(0);"  key={media.id_str} target='_blank'>
+            <img id ="tweet_mediaID"className='tweet_media' onClick={() => {this.openImageInWindow(media)}} src={media.media_url} />
           </a>
         );
       } else {
         return '';
       }
     });
   }
+  openImageInWindow (med) {
```

Comment:
Nocturn uses Redux to split application logic from view logic. Please move this function to tweet-container.js.